src/components/htmlvideoplayer/plugin.js

```diff
@@ -556,6 +558,15 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
             setCurrentTrackElement(index);
         };
 
+        self.resetSubtitleOffset = function() {
+            if (currentTrackOffset) {
```

Comment:
Is it necessary to check this? I think this should be fine?
```
            currentTrackOffset = 0;
            showTrackOffset = false;
```

Reply:
You are right, I will add a commit.